### PR TITLE
Junseok / apply chexmix from private repo

### PR DESCRIPTION
**Pull request by elephantoid**
Summary:
  - apply `src/chexmix/` from private repo 
    - add
      - tables, graphs, unittest
    - modify
      - data, datasources, visualization
    - remove
      - `excel.py`, `graph.py` -> `graph/`
  - add tutorial notebooks about classyfire and chexmix
  - gitignore `.idea`

### CODE REVIEW

File: src/chexmix/datasources/base.py [hunk 1]
```diff
@@ -40,13 +40,17 @@ def add_total_pmids(node):
     return total_pmids
 
 
-def add_pmids(node_table, annotation_table, prefix, root):
-    prefix_len = len(prefix)
-
+def convert_pmid(node_table, root):
```
Comment (pbj8723) [inline src/chexmix/datasources/base.py:43]: convert라는 naming이 맞는지애매한것같습니다

File: src/chexmix/graph/mesh.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    def get_mesh_node_id_from(bioentity: str):
+        """Get node id.
+
+        :param bioentity: mesh bio entity
+        :return: node id
+        """
+        if bioentity[0] == "D":
+            node_id = HierarchicalGraph.create_node_id(Header.MeSHD, bioentity)
+        else:
+            node_id = HierarchicalGraph.create_node_id(Header.MeSHC, bioentity)
```
Comment (pbj8723) [inline src/chexmix/graph/mesh.py:30]: one line으로 될것같긴하네요

**Review by pbj8723 — APPROVED** [review]
lgtm %c

**Review by hojinyoo — APPROVED** [review]
lgtm

**Pull request merged**